ckanext/ontario_theme/fanstatic/common.css

```diff
@@ -2450,25 +2450,24 @@ screen and (min-width: 96em) {
 /* ========================================================================
    About and Help page - On this page
    ======================================================================== */
-ol.table-of-contents {
+ol.ontario-page-navigation-list {
   list-style-position: outside;
   margin-left: 1.5em;
   padding-left: 0;
+  list-style-type: disc;
 }
 
-ul.table-of-contents {
+ul.ontario-page-navigation-list {
   list-style-type: none;
   margin-left: 0;
   padding: 0;
 }
 
-.table-of-contents {
+.ontario-page-navigation-list {
   margin-bottom: 1.25rem;
-  columns: 2 21rem;
-  column-gap: 2.5rem;
 }
 
-.table-of-contents li {
+.ontario-page-navigation-list li {
   break-inside: avoid;
   padding: 0 0 0.75rem 0.25rem;
 }
```

Comment:
Rename this section "In-page navigation - Design System" and you can remove all the css styling for the in-page navigation. Copy and paste the formatted css in this section instead.